graphics/xfractint/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                xfractint
+version             20.04p14
+categories          graphics
+maintainers         @jflude \
+                    openmaintainer
+description         A highly versatile fractal generator
+long_description    Fractint is a freeware fractal generator created for PCs \
+    and ported to Unix/X Window. It has been in development since 1988. \
+    Fractint can also draw cellular automata and stereoscopic images.
+homepage            https://www.fractint.org/
+platforms           darwin
+license             Freeware
+
+master_sites        https://www.fractint.org/ftp/current/linux/
+checksums           md5 0e997a83e92fd1119d09a97351f9f788
+
+patch.args          -p1
+patchfiles          patch-prefix-x11-dirs.diff
+
+post-patch {
+    reinplace "s|__PREFIX__|${prefix}|" "${worksrcpath}/Makefile"
+}
+
+use_configure       no
```

Comment:
Because there is no configure file, the universal variant won't work. Either disable the universal variant or make sure it works properly. Please see the wiki page about this. 

https://trac.macports.org/wiki/UsingTheRightCompiler

Reply:
Supplied CC, CXX & CPP defs in the Portfile, removed CC def from Makefile (strictly speaking, unnecessary, but for clarity), confirmed build uses only the MacPorts defs.